src/components/GoogleLogin/GoogleLogin.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { GoogleLogin } from 'react-google-login'
+import API from '../../Services/axios';
+import { useDispatch } from 'react-redux';
+import { login } from '../../redux/'
+import { URLS } from '../../urls'
+import { useHistory } from 'react-router-dom';
+import { FormInput } from '../../utils/styles'
+
+export const LoginWithGoogle = () => {
+  const history = useHistory()
+  const dispatch = useDispatch();
+  const successResponseGoogle = response => {
+    API.post(`users/google/`, { token: response.accessToken })
```

Comment:
we need to make a generic APICaller and use that everywhere instead of calling the axios.